### PR TITLE
Developer Tutorial Move - 1

### PR DESCRIPTION
**Pull request by kb1lqd**
# Updates

Python_Developers_Tutorial folder contains:
* "foundation" folder
* "miscellaneous" folder

Removed folders:
* 1-Basic_Proxy_Interactions_And_Programming
* 2-Advanced_Proxy_Programs

Renamed Folders
* Simply removed the numbers prefixed to the folder names of the tutorials in the
foundation/miscellaneous folders.

Other:
* Update Readme.md to latest
* Fixed basic spelling typo's in readme.md
* Moved Readme.md from "Tutorials" folder to "Python_Developer_Tutorials" folder

### CODE REVIEW

File: Tutorials/Python_Developer_Tutorials/Readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+# Faraday Python Developer Tutorials
+
+This tutorial series is focused on introducing Faraday as a digital radio platform to program using Python. The reader will begin by learning fundamental transmissions of commands to both local and remote (using RF) devices. The tutorial series will conclude by building upon each skill and tool-set learned to develop a basic radio data  transmission program capable of transmitting data, text, etc... of arbitrary size from one Faraday to another through wireless transmissions.
```
Comment (kb1lqc) [inline Tutorials/Python_Developer_Tutorials/Readme.md]: ## Non-Blocking
* Spelling: http://www.dictionary.com/browse/toolset
* Wording "The tutorial series will conclude by building upon each skill and tool-set learned to develop a basic radio data  transmission program capable of transmitting data, text, etc... of arbitrary size from one Faraday to another through wireless transmissions." for example uses transmission or transmissions a lot which sounds weird.

File: Tutorials/Python_Developer_Tutorials/Readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+**Key Concepts**
+
+* Digital Communications Fundamentals
+  * Frame/packet, error detection, fragmentation, encapsulation, ARQ (automatic retry-request)
+* Programming with Faraday tool-sets/modules
```
Comment (kb1lqc) [inline Tutorials/Python_Developer_Tutorials/Readme.md]: ## Non-Blocking
* Spelling: http://www.dictionary.com/browse/toolset

File: Tutorials/Python_Developer_Tutorials/Readme.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* Building more advanced Faraday host computer programs
+
+> These tutorials rely on the currently implemented commands/functionality of Faraday and thus provide a great platform for experimentation. Higher data throughput and functionality can be achieved using other tool-sets or in conjunction with more specialized firmware programming that will be covered at a later date.
```
Comment (kb1lqc) [inline Tutorials/Python_Developer_Tutorials/Readme.md]: ## Non-Blocking
* Spelling: http://www.dictionary.com/browse/toolset

File: Tutorials/Python_Developer_Tutorials/Readme.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+##Introduction To Faraday Proxy Programming
```
Comment (kb1lqc) [inline Tutorials/Python_Developer_Tutorials/Readme.md]: ## Blocking
* I would label what is complete and what is not with a little note i.e

**[Commanding - Local](foundation/Commanding-Local/)** (Outdated)

**Review by kb1lqc — CHANGES_REQUESTED** [review]
One simple blocking change that you should note what isn't updated so when this is merged people know what is old/stale/outdated on the readme.md

File: Tutorials/Python_Developer_Tutorials/Readme.md (hunk 1 above)
Comment (kb1lqd) [reply Tutorials/Python_Developer_Tutorials/Readme.md]: Fixed typo, reworded.

File: Tutorials/Python_Developer_Tutorials/Readme.md (hunk 2 above)
Comment (kb1lqd) [reply Tutorials/Python_Developer_Tutorials/Readme.md]: Fixed.

File: Tutorials/Python_Developer_Tutorials/Readme.md (hunk 4 above)
Comment (kb1lqd) [reply Tutorials/Python_Developer_Tutorials/Readme.md]: Added notes.

**Comment by kb1lqd** [comment]
@kb1lqc updates performed per review, ready for re-review.

File: Tutorials/Python_Developer_Tutorials/Readme.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+
+# Faraday Python Developer Tutorials
+
+This tutorial series is focused on introducing Faraday as a digital radio platform. The reader will begin by learning fundamental commands used to used to both communicate with a locally connected Faraday as well as commands used to control and transfer data to remote Faraday radios using wireless transmissions. Each tutorial series builds upon the prior to create a simple wireless transmission program capable of transmitting both text and arbitrary data from one Faraday to another.
```
Comment (kb1lqc) [inline Tutorials/Python_Developer_Tutorials/Readme.md]: "...commands used to used to both communicate..." makes no sense

File: Tutorials/Python_Developer_Tutorials/Readme.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  * How to send commands to a remote Faraday device over RF.
+* **[Experimental RF Packet Forwarding](foundation/RF-Transmit-Receive-Packet) (OUTDATED - UPDATE IN PROGRESS)**
+  * An introduction to using the simple RF packet forward command that sends a local UART payload over RF to a specified remote Faraday device. This is the basis for more advanced programs such as messaging and data/file transfers.
+
+## Building Fundamental Applications
+
+* **[Simple Text Messaging](foundation/Simple_Text_Messaging/) (OUTDATED - UPDATE IN PROGRESS)**
+  * Utilizing the experimental RF packet forwarding command to build a simple text messaging application.
+* **[Simple Text Messaging - Creating Functional Objects](foundation/Simple_Text_Messaging_Creating_Objects/) (OUTDATED - UPDATE IN PROGRESS)**
+  * This example tutorial implements the "Simple Text Messaging" application into a generic transmitter/receiver python object. This is a major step towards implemented a RESTful FLASK based application interface.
```
Comment (kb1lqc) [inline Tutorials/Python_Developer_Tutorials/Readme.md]: - implemented should be "implementing"
- FLASK should be Flask

**Review by kb1lqc — CHANGES_REQUESTED** [review]
Wording is off and a tad confusing. Fix those and it should be good.

File: Tutorials/Python_Developer_Tutorials/Readme.md (hunk 5 above)
Comment (kb1lqd) [reply Tutorials/Python_Developer_Tutorials/Readme.md]: Fixed.

File: Tutorials/Python_Developer_Tutorials/Readme.md (hunk 6 above)
Comment (kb1lqd) [reply Tutorials/Python_Developer_Tutorials/Readme.md]: Fixed.

**Review by kb1lqc — APPROVED** [review]
Thanks for updating the wording. Looks good.

**Pull request merged**